content/ri/using-redisinsight/api.md

```diff
@@ -0,0 +1,178 @@
+---
+Title: RedisInsight REST API
```

Comment:
Not sure if we want to call this REST API.  I know it's a REST api, but this implies some kind of contract.  I'd rather have a title that reflects the ability to add databases in an automated way and than in the introduction state that it can be done via the rest API.  for now at least until we see more REST api use cases.  This might also change slightly if we go into our 4 editions we spoke about @nishantgeorge

Reply:
Changed the title and added an intro. Works?